### ISSUE TITLE
FireFox Issues

### ISSUE THREAD

**Issue by PR0SAT** [andreruffert/rangeslider.js#324]
First of all, Andre thx for this great piece of java scripting. Now the sad part. In FireFox recent version (satanic 66.0.1) the bar gauge stoped moving, no errors or warning in console. Other browesers (IE,Chrome) are just fine. Hope this will be easy to fix.

On the screen lef side IE on right FF with drag control element blocked.
![1](https://user-images.githubusercontent.com/33123454/54923177-01cd2680-4f0a-11e9-8d12-ec1ed6460180.png)

Cheers!

**Comment by andreruffert** [comment]
I don't think this is a Firefox issue. For me it works like expected. 
To me this sounds more like the initialization is not happening properly.
Are you initializing the plugin on DOM ready?

![Screen Shot 2019-03-25 at 15 38 51](https://user-images.githubusercontent.com/464300/54928399-23331000-4f14-11e9-850c-da8598c2cffa.png)

**Comment by tiesont** [comment]
I can confirm that this works as expected in 66.0.1.

**Comment by PR0SAT** [comment]
You are all right! It's working, It's just me loading the rangeslider and it's css,js dynamically and firefox weired DOM behaviour insertion. The approach with Mutation Observer solved my issue:
`
var mutationObserver = new MutationObserver(function(mutations) {
  mutations.forEach(function(mutation) {});
		if (!$('.rangeslider').length) {
alert('afterShow');
			$.getScript('//' + window.location.host + '/js/rangeslider.js-2.3.0/rangeslider.min.js').done(function(){
				var r = $('input[type="range"]');
				// Initialize
				r.each(function(){
					$(this).rangeslider(r_settings);
				});
			});
		}
});
var rc = $('#range-slider-container')[0];
mutationObserver.observe(rc, {
  attributes: true,
  characterData: true,
  childList: true,
  subtree: true,
  attributeOldValue: true,
  characterDataOldValue: true
});`

Sorry for the false positive.

**Comment by bencergazda** [comment]
@PR0SAT @andreruffert I was having the same issue when using https://github.com/filamentgroup/loadJS and https://github.com/filamentgroup/loadCSS. Replacing it with https://github.com/muicss/loadjs solved the problem. I suggest rather using using this latter library for aync loading scripts.
